lldb/source/Plugins/TypeSystem/Clang/TypeSystemClang.cpp

```diff
@@ -1156,6 +1159,10 @@ CompilerType TypeSystemClang::GetTypeForDecl(ObjCInterfaceDecl *decl) {
   return GetType(getASTContext().getObjCInterfaceType(decl));
 }
 
+CompilerType TypeSystemClang::GetTypeForDecl(clang::ValueDecl *value_decl) {
```

Comment:
Could you remind me where the need for `ValueDecl` comes from?

Reply:
ValueDecl is a base of VarDecl. For this patch I only need to know the types of `VarDecl`s, but I figured I could use the lowest class that does the trick (defines `getType`), in case that's useful in the future.

Reply:
Ahh makes sense, didn't look at the inheritance hierarchy carefully enough